From 0dce59bb4edc9c10ab2baf8d6d33cb3f204fc1f3 Mon Sep 17 00:00:00 2001
Message-Id: <0dce59bb4edc9c10ab2baf8d6d33cb3f204fc1f3.1656411268.git.git@grubix.eu>
From: Michael J Gruber <git@grubix.eu>
Date: Tue, 28 Jun 2022 11:53:43 +0200
Subject: [PATCH] doc: provide nmbug link for notmuch-git info page

8ed68c1b ("CLI/git: change defaults for repo and prefix", 2022-05-07)
introduced an nmbug symlink for the notmuch-git man page. Do the same
for the info page.

Signed-off-by: Michael J Gruber <git@grubix.eu>
---
 doc/Makefile.local | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/doc/Makefile.local b/doc/Makefile.local
index 2f67f4de..1908eb83 100644
--- a/doc/Makefile.local
+++ b/doc/Makefile.local
@@ -148,7 +148,9 @@ else
 install-info: build-info
 	mkdir -m0755 -p "$(DESTDIR)$(infodir)"
 	install -m0644 $(INFO_INFO_FILES) $(DESTDIR)$(infodir)
-	for file in $(INFO_INFO_FILES); do install-info $$file $(DESTDIR)$(infodir)/dir; done
+	cd $(DESTDIR)$(infodir) && ln -sf notmuch-git.info nmbug.info
+	for file in $(INFO_INFO_FILES) $(DESTDIR)$(infodir)/nmbug.info; do install-info $$file $(DESTDIR)$(infodir)/dir; done
+	cd 
 endif
 
 $(dir)/config.dox: version.stamp
-- 
2.37.0.rc2.141.g1312637e40

